web/src/main.tsx

```diff
@@ -45,7 +45,7 @@ const queryClient = new QueryClient({
       retry: MAX_RETRIES,
       refetchOnWindowFocus: false,
       // by default data is cached and valid forever, as we handle invalidation ourselves
-      cacheTime: Number.POSITIVE_INFINITY,
+      gcTime: Number.POSITIVE_INFINITY,
```

Comment:
I'm actually not sure we should disable garbage collection here, as I think that means the cache (garbage) grows indefinitely?

What was the original reasoning for this?

Reply:
Hmm, as far as I remember it is by default 5 min and it would refetch data files every 5 min which seemed very wasteful given that nothing will change within the same hour. This is still the same situation with Cloudflare cache, as we don't want the client to redownload data every 5 min.

And since we automatically invalidate at the switch of the hour, it also isn't useful in any way :)

I can't remember how it actually works with the garbage collection part... But since we will overwrite/fetch fresh data on every new session, will that not "overwrite" the old one?

Reply:
I have no idea if it will overwrite it or save a reference to the old data. I know there are some features around tanstack router that allows you to serve the "old" data in case the request fails or temporarily while it's fetching the new stuff.

I'll have to dig into the details but should be fine to leave as is for now.